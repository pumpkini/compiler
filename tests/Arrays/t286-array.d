int main(){
    int i;
    int[] a;
    int len;

    len = 5;
    a = NewArray(len, int);
    a[0] = 1;
    a[1] = 2;
    a[2] = a[1];
    a[1] = 10;
    a[3] = 4;
    a[4] = 5;
    for(i = 0; i < len; i = i+1){
        Print(a[i]);
    }
    Print(a.length());
}