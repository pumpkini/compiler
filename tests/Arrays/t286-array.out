1
10
2
4
5
5
